apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -191,6 +191,7 @@ class TopInstructions extends Component {
           })
       );
     }
+
```

Comment:
nit: remove this blank line?

Reply:
I guess no longer related to this PR to have changes to this file, but there are spaces between each if block so it seems more consistent 🤷